### PR TITLE
Bug 1986003: Bump k8s.io to 1.22.0-rc.0

### PR DESCRIPTION
**Pull request by soltysh**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -81,8 +81,6 @@ clean:
 export TP_DEPLOYMENT_YAML ?=./manifests/0000_20_kube-apiserver-operator_06_deployment.yaml
 export TP_CMD_PATH ?=./cmd/cluster-kube-apiserver-operator
 
-$(call verify-golang-versions,Dockerfile.rhel7)
```
Comment (sttts) [inline Makefile:84]: where is this duplicated?

Comment (soltysh) [reply Makefile:84]: we already have that done inside build-machinery-go https://github.com/openshift/build-machinery-go/blob/master/make/golang.mk#L11

Comment (p0lyn0mial) [reply Makefile:84]: line 37 - https://github.com/openshift/cluster-kube-apiserver-operator/blob/22f2a00c6293c3a979804651e4446a57ec331ed6/Makefile#L37

**Pull request merged**